.github/workflows/actions.yml

```diff
@@ -23,6 +23,8 @@ jobs:
         uses: actions/checkout@v2
       - name: Run Go Security
         uses: securego/gosec@master
+        with:
```

Comment:
Can we move this `gosec` check to the common actions level instead of adding in each of the drivers?

Reply:
This was discussed in the PO meeting and was concluded that we will go ahead with master branch, so that we don't miss out any newly added checks.

Reply:
As agreed offline, go ahead with the merge.